wgpu/src/util/mod.rs

```diff
@@ -64,6 +66,13 @@ pub fn make_spirv_raw(data: &[u8]) -> Cow<[u32]> {
         Cow::from(words)
     };
 
+    // swap if that corrects the magic
```

Comment:
Could this be a bit more verbose

Reply:
fair point, i don't really know how to phrase things well :)